.settings/org.eclipse.jdt.core.prefs

```diff
@@ -1,9 +1,15 @@
 eclipse.preferences.version=1
-org.eclipse.jdt.core.compiler.codegen.targetPlatform=12
-org.eclipse.jdt.core.compiler.compliance=12
```

Comment:
The changes here will not be added to the final PR. It is the environment I am currently working with...should not be submitted to the master branch.